graphics/tinygl/zbuffer.cpp

```diff
@@ -170,17 +170,65 @@ void FrameBuffer::clearRegion(int x, int y, int w, int h, int clearZ, int z, int
 
 void FrameBuffer::blitOffscreenBuffer(Buffer *buf) {
 	// TODO: could be faster, probably.
+#define UNROLL_blit(_a) {\
```

Comment:
No macros, please :). We moved away in tinygl. They are hard to debug. Maybe we could use templates.